### ISSUE TITLE
Burning in Subtitles No Longer Working

### ISSUE THREAD

**Issue by MuhChicken** [RandomEngy/VidCoder#1085]
I've used VidCoder in the past to encode subtitle tracks into anime videos, however it has stopped working properly.  A couple of days ago I encoded 24 episodes and chose to burn in sub track 2.  However, upon watching the videos the subs are not burned in and are only still available to be selected.  I also checked the output videos using VLC Media Player and the sub tracks are available there and not burned in as well.  This was not the previous behavior of this app combined with Handbrake.  In the past I just used the picker to select my preset to burn in sub track 1 or 2 and then dragged and dropped the folder containing the videos into VidCoder.  After encoding I checked them with VLC and the sub track options were grayed out because they had been burned in and were no longer available for selection.

Both apps are up to date as of today according to the in-app information.
VidCoder = 7.15
Handbrake = 1.5.1

We use a Roku to play media from Plex, but the Roku does not support PGS subtitles. Because of this, I have been using VidCoder to burn them in.  Prior to encoding I check several episodes to verify the correct sub track for my needs.  Sub track 1 is generally for full subtitles on videos that have not yet been dubbed in English and sub track 2 is generally for videos that have been dubbed in English, but still have foreign language text and some foreign audio content such as theme songs and the like.

Note: I setup my encoding processes and saved them so I could use the Picker a long time ago. I'm wondering if I need to setup new processes from scratch or if there is something I can do to the existing processes to make them actually burn the sub tracks into the video like they are supposed to.  I just tried encoding a single video with the picker and it failed to burn the subs in, so I will include a screen shot showing the sub track settings in case it helps.
![Vidcoder Sub Track 2 Burn In](https://user-images.githubusercontent.com/120859231/208271768-0ead569b-596a-46e1-b686-72774359ced6.jpg)

**Comment by MuhChicken** [comment]
I have edited the file/folder names and attached the logs for 2 separate videos that were in the same queue, added at the same time, and supposed to be encoded with the same settings.  It looks like the first video encoded the subs properly, but the other one did not and just left the one subtitle track available in the final output.  Hopefully this can help you determine what went wrong.  The file names are either worked for the scan and encode that worked or did not work for the one that failed to encode the subtitles.

I also uninstalled and reinstalled VidCoder and setup a new custom picker that was supposed to select sub track 2 and burn it in.  It still failed even when only encoding the 1 video.

Here are the older logs from when I ran the batch encode a couple of days ago in case it helps:

[2022-12-14 02.53.58 Encode did not work.mkv-succeeded.txt](https://github.com/RandomEngy/VidCoder/files/10253032/2022-12-14.02.53.58.Encode.did.not.work.mkv-succeeded.txt)
[2022-12-14 00.14.51 Scan worked.txt](https://github.com/RandomEngy/VidCoder/files/10253033/2022-12-14.00.14.51.Scan.worked.txt)
[2022-12-14 00.15.18 Scan did not work.txt](https://github.com/RandomEngy/VidCoder/files/10253034/2022-12-14.00.15.18.Scan.did.not.work.txt)
[2022-12-14 00.19.22 Encode worked.mkv-succeeded.txt](https://github.com/RandomEngy/VidCoder/files/10253035/2022-12-14.00.19.22.Encode.worked.mkv-succeeded.txt)
